### PR TITLE
LPD-30899 Reorder class usages

### PR DESCRIPTION
**Pull request by kyle-miho**
https://liferay.atlassian.net/browse/LPD-30899
cc: @nicolas-sss @drewbrokke 
Should be easy to review, simple bug / change

### CODE REVIEW

**Comment by kyle-miho** [comment]
image
Passed Manual Testing locally following the steps in the Jira Ticket

**Comment by kyle-miho** [comment]
ci:test:sf

File: modules/util/source-formatter/src/main/resources/dependencies/replacements.json [hunk 1]
```diff
@@ -3904,19 +3904,25 @@
...
 		"classNames": [
 			"CommerceOrderItemLocalService",
-			"CommerceOrderItemService"
+			"CommerceOrderItemLocalServiceUtil"
 		],
 		"from": "addCommerceOrderItem(long paramName, long paramName, int paramName, int paramName, String paramName, CommerceContext paramName, ServiceContext paramName)",
 		"issueKey": "LPS-197459",
+		"newImports": [
+			"com.liferay.portal.kernel.util.PortalUtil"
+		],
```
Comment (nicolas-sss) [inline modules/util/source-formatter/src/main/resources/dependencies/replacements.json:3913]: @kyle-miho shouldn't be `BigDecimal`?

**Review by nicolas-sss — CHANGES_REQUESTED** [review]

File: modules/util/source-formatter/src/main/resources/dependencies/replacements.json (hunk 1 above)
Comment (kyle-miho) [reply modules/util/source-formatter/src/main/resources/dependencies/replacements.json]: @nicolas-sss Oops yea my bad, fixed!

**Review by drewbrokke — APPROVED** [review]

**Pull request closed without merging**